Bottom: fdd492cdb13406bafcf14c37b554b2de76a99d5b
Top:    7cfa8b8bbb7449eac973ab817f043b3f1a3850b7
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-08-07 13:53:54 -0700

Refresh of dup2

---

diff --git a/src/preload.c b/src/preload.c
index 085e8b4..b18d310 100644
--- a/src/preload.c
+++ b/src/preload.c
@@ -921,6 +921,7 @@ int dup2(int oldfd, int newfd)
 	struct fd_info *oldfdi, *newfdi;
 	int ret;
 
+	init_preload();
 	oldfdi = idm_lookup(&idm, oldfd);
 	if (oldfdi && oldfdi->type == fd_fork)
 		fork_passive(oldfd);
